### PR TITLE
Issue 73/add ceti to partners

### LINKED ISSUE #73: Enhancement: [Add CETI to partners ]
### Describe the Current Behavior/Feature:
Add CETI to partners as a secondary partner in secondaryPartnerList.js mapping from seconardaryPartners.js should populate it onto page.
https://ceti.institute/

---

### Proposed Implementation:

Do whatever it takes!

### PR DESCRIPTION
**Pull request by andycwilliams**
## This PR:

Resolves #73 

Adds CETI to the list of secondary partners. Looks out of whack because #78 isn't merged yet. Once that's done this will look better. Additionally added a testimonial/summary for Epicodus.

Also, I tried organizing the image assets a bit more. The only difference should be that they're in subfolders now. Yet they render differently on the Projects page. Although in the Projects Brief section they seem unchanged. 

That was outside of the scope for this PR, so I put all that into separate commits and they should be easily undone. Hence leaving this as a draft for now.

---

## Screenshots (if applicable):

---

## Future Steps/PRs Needed to Finish This Work (optional):

Merge the secondary partner styling updates.

Decide whether it's worth dividing the images into subfolders.

### CODE REVIEW

File: src/components/home/partners/secondaryPartnerList.js [hunk 1]
```diff
@@ -15,14 +15,21 @@ export const secondaryPartnerList = [
...
-    partnerLogo: '/assets/ordsnLogo.png',
+    partnerLogo: '/assets/partnerImages/ordsnLogo.png',
     website: 'https://www.ordsn.org/'
   },
   {
     company: 'Epicodus',
     testimonial:
-      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam, quaerat saepe possimus ex dolor deleniti excepturi alias, est in tempore similique, inventore doloribus voluptatibus voluptate. Reiciendis minima nihil et dicta.',
-    partnerLogo: '/assets/epicodusLogo.png',
+      "Epicodus's mission is to help people learn the skills they need to get great jobs. For us, “great jobs” means jobs in growing industries that pay well and provide rewarding work. Beyond the particular skills needed to get these jobs, we aim to help our students become confident self-teachers who can adapt to changing job markets, and great communicators who will work well in teams. We focus on serving people who, by birth or circumstance, don't have easy access to learning the skills they need to get great jobs..",
```
Comment (xscottxbrownx) [inline src/components/home/partners/secondaryPartnerList.js]: kill the extra period at end of paragraph

File: src/components/home/partners/secondaryPartnerList.js [hunk 2]
```diff
@@ -15,14 +15,21 @@ export const secondaryPartnerList = [
...
+    partnerLogo: '/assets/partnerImages/epicodusLogo.png',
     website: 'https://www.epicodus.com/'
+  },
+  {
+    company: 'CETI',
+    testimonial:
+      'A Creative and Emergent Technology Institute, CETI champions lifelong learning and collaborative, interdisciplinary innovation in creative and emergent technologies',
```
Comment (xscottxbrownx) [inline src/components/home/partners/secondaryPartnerList.js]: I'd prefer either the **Mission** section of their `About` page:
```
CETI champions lifelong learning and collaborative, interdisciplinary innovation in creative and emergent technologies. Through radically inclusive community participation, we rapidly build expertise for an astonishing diverse group of people. CETI contends that our most urgent challenges can be imaginatively addressed through art, storytelling and invention.
```

OR

the **Who We Are** section of their `About` page:
```
CETI prioritizes including and collaborating with communities previously marginalized, under-represented, and under-served  in tech. Through our lived experiences and diversity in demographics, we bring a diversity of critical and cultural lenses to the intentional design of our programs, as well as a laser-focus on inclusion and support of communities significantly under-represented in technology conversations.  

CETI is a collective—of artists, technologists, storytellers, educators, researchers, innovators, and leaders—composed of experts and lifelong learners, excited to share our ideas and skills. A subset of us engage with each project, alternatively taking on roles as learners, facilitators, creators, producers, and advisors, wielding the powerful tools of art, emerging technology, and storytelling  to imagine and create inclusive, just futures.
```

**Review by xscottxbrownx — CHANGES_REQUESTED** [review]

File: src/components/home/partners/secondaryPartnerList.js (hunk 2 above)
Comment (andycwilliams) [reply src/components/home/partners/secondaryPartnerList.js]: The first one is more concise and to-the-point imo. I vote that one.

**Review by ogorman89 — APPROVED** [review]
Looks good. I think once we get this and #78 merged we should give it one more look for adjustments.

**Review by Jared-Krajewski — CHANGES_REQUESTED** [review]
The folder changes aren't really part of this issue so not sure how that fell into this PR.  That seems more of a part of #56 either way, but make sure it doesn't cause issues with that PR since Ian is currently doing that. I am thinking all of the image changes should be reverted and done in that PR.

On the note of separating the images, if we are going to separate them it should be a bit more specific instead of `codepdxImages`. maybe `partnerLogos` for the partner logos, `projectLogos` for project logos ..ect. Also if we are putting them all into folders, we should put them all into folders not just some of them. 

The changes regarding issue #73 are fine to me otherwise.

**Review by Jared-Krajewski — APPROVED** [review]

**Comment by andycwilliams** [comment]
@xscottxbrownx Once you approve we can get this merged

**Review by xscottxbrownx — APPROVED** [review]

**Pull request merged**